java-vtl-script/src/main/java/no/ssb/vtl/script/functions/VTLTrunc.java

```diff
@@ -0,0 +1,60 @@
+package no.ssb.vtl.script.functions;
+
+/*-
+ * ========================LICENSE_START=================================
+ * Java VTL
+ * %%
+ * Copyright (C) 2016 - 2017 Arild Johan Takvam-Borge
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+import com.google.common.annotations.VisibleForTesting;
+import no.ssb.vtl.model.VTLNumber;
+import no.ssb.vtl.model.VTLObject;
+
+import java.math.BigDecimal;
+
+public class VTLTrunc extends AbstractVTLFunction<Number> {
+
+    private static final Argument<VTLNumber> DS = new Argument<>("ds", VTLNumber.class);
+    private static final Argument<VTLNumber> DECIMALS = new Argument<>("decimals", VTLNumber.class);
+
+    @VisibleForTesting
+    VTLTrunc() {
+        super("trunc", Number.class, DS, DECIMALS);
+    }
+
+    @Override
+    protected VTLObject<Number> safeInvoke(TypeSafeArguments arguments) {
+
+        VTLNumber ds = arguments.get(DS);
+        VTLNumber decimals = arguments.get(DECIMALS);
+
+        if (ds.get() == null) {
+            return VTLObject.of((Number)null);
+        }
+        if (decimals.get() == null) {
+            decimals = VTLNumber.of(0);
+        }
+        if (decimals.get().intValue() < 0) {
+            throw new IllegalArgumentException("Number of decimals must be equal to or greater than zero");
+        }
+
+        BigDecimal bigDecimal = new BigDecimal(ds.get().toString());
```

Comment:
I think `BigDecimal.valueOf(1.0D)`should be used here.

http://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html#valueOf-double-